### PR TITLE
better error log for container environment error

### PR DESCRIPTION
**Pull request by trapacska**
Screenshot 2020-02-20 at 13 19 31
became
Screenshot 2020-02-20 at 13 29 05

### CODE REVIEW

**Comment by trapacska** [comment]
this feature looks working fine (specifying environment container)

File: main.go [hunk 1]
```diff
@@ -547,7 +548,8 @@ is available in the $BITRISE_XCODE_RAW_RESULT_TEXT_PATH environment variable`)
 				if exportMethod == exportoptions.MethodAppStore {
 					iCloudContainerEnvironment = "Production"
 				} else if cfg.ICloudContainerEnvironment == "" {
-					fail("project uses CloudKit, but iCloud container environment input not specified")
+					fail("Your xcode project uses CloudKit capability and the export method specified is not app-store.\n" +
+						"In this case you have to specify iCloud container environment (Development or Production) in ICloudContainerEnvironment step input field.")
```
Comment (lpusok) [inline main.go]: Would modify to be more user friendly: 
in the "iCloud container environment" step input field.

File: main.go [hunk 2]
```diff
@@ -547,7 +548,8 @@ is available in the $BITRISE_XCODE_RAW_RESULT_TEXT_PATH environment variable`)
 				if exportMethod == exportoptions.MethodAppStore {
 					iCloudContainerEnvironment = "Production"
 				} else if cfg.ICloudContainerEnvironment == "" {
-					fail("project uses CloudKit, but iCloud container environment input not specified")
+					fail("Your xcode project uses CloudKit capability and the export method specified is not app-store.\n" +
```
Comment (lpusok) [inline main.go]: "iCloud container environment input not specified" would include the root cause of the error like this:

Your project uses CloudKit and iCloud container environment input not specified. Export method is: %s (For app-store export method Production container environment is implied.)

File: main.go [hunk 3]
```diff
@@ -547,7 +548,8 @@ is available in the $BITRISE_XCODE_RAW_RESULT_TEXT_PATH environment variable`)
 				if exportMethod == exportoptions.MethodAppStore {
 					iCloudContainerEnvironment = "Production"
 				} else if cfg.ICloudContainerEnvironment == "" {
-					fail("project uses CloudKit, but iCloud container environment input not specified")
+					fail("Your project uses CloudKit and \"iCloud container environment\" input not specified.\n"+
+						"Export method is: %s (For app-store export method Production container environment is implied.)", exportMethod)
```
Comment (godrei) [inline main.go]: typo `implied`

Comment (godrei) [reply main.go]: I would log:

Your project uses CloudKit **but** \"iCloud container environment\" input not specified.
The export method is: %s (For app-store export method Production container environment is automatically set.)

**Pull request merged**